example/sample.ts

```diff
@@ -34,12 +34,8 @@ const IMAGE_PATH = '/samantha-gades-BlIhVfXbi9s-unsplash.jpg';
 const state = {
   defaultDevices: new Map<MediaDeviceKind, string>(),
   bitrateInterval: undefined as any,
-  blur: BackgroundBlur(BLUR_RADIUS, undefined, (stats) => {
-    // console.log('frame processing stats', stats);
-  }),
-  virtualBackground: VirtualBackground(IMAGE_PATH, undefined, (stats) => {
-    // console.log('frame processing stats', stats);
-  }),
+  backgroundMode: null as NonNullable<BackgroundProcessorOptions['mode']> | null,
+  background: BackgroundProcessor({ mode: 'background-blur', blurRadius: BLUR_RADIUS }),
```

Comment:
Nit
```suggestion
  backgroundProcessor: BackgroundProcessor({ mode: 'background-blur', blurRadius: BLUR_RADIUS }),
```

Reply:
💯 Not sure how I let that slip through, thanks!